packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -29,32 +29,42 @@ import { useRef, useState } from '@wordpress/element';
  * Internal dependencies
  */
 import { getMoversSetup } from '../block-mover/mover-description';
+import BlockTransformationsMenu from '../block-switcher/block-transformations-menu';
 
 const BlockActionsMenu = ( {
-	onDelete,
```

Comment:
NOTE HERE that I have reordered the props for better clarity. If people hate it I can change it back, but I find it's a lot easier to understand myself when I organize it this way.
1. Props are now organized by either coming from `withDispatch`, `withSelect`, or passed in when created.
2. Props are alphabetically organized within each section.